src/App.tsx

```diff
@@ -1,9 +1,17 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const checkQueryMatch = (movie_param: string) => (
+    movie_param.toLocaleLowerCase().includes(query.toLowerCase().trim()));
```

Comment:
```suggestion
  const checkQueryMatch = (movieParam: string) => (
    movieParam.toLocaleLowerCase().includes(query.toLowerCase().trim()));
```
You are using camelCase everywhere. What's the purpose of adding underline between words here?